docs/modules/ROOT/pages/how-tos/testing/integration/debugging.adoc

```diff
@@ -0,0 +1,44 @@
+= Debugging Integration Tests
+
+In the event of an integration test scenario failing, there are a number of steps to take in order to debug the issue before rerunning the test in question. These steps are outlined below.
+
+
+.Prerequisites
+- You have a snapshot that has completed all its initial tests. All initial tests must be finished before attempting to debug the failed test.
```

Comment:
What do you mean by "initial tests"? Do you mean the scanners and linters that run as a part of the build pipeline?

I guess "initial tests" is the language used in `docs/modules/ROOT/pages/how-tos/testing/integration/rerunning.adoc`. It's not quite clear to me, but it's probably better to be consistent here than anything else. If we were to revise it, let's do that in a separate PR.

Reply:
Hi Ralph, I was just adopting the language from the rerunning page and yeah it was my understanding that this was referring to the build pipeline checks!

Reply:
We use that term consistently outside of the integration test pages.

Reply:
This is now reflected in the latest commit.